addons/account_payment/models/account_payment.py

```diff
@@ -55,7 +55,7 @@ def _compute_amount_available_for_refund(self):
             )
             if (
                 tx_sudo.provider_id.support_refund
```

Comment:
With both payment providers and payment methods having their `support_refund` field accept the value `none`, you should never test for just `if [...].support_refund`.

Reply:
Got it :100: